repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/yql/YqlOrderBy.java

```diff
@@ -146,6 +146,15 @@ public String toString() {
         return keys.isEmpty() ? "unordered" : format("order by %s", keys.stream().map(Object::toString).collect(joining(", ")));
     }
 
+    /**
+     * Sort order for {@code ORDER BY} clause.
+     * <p><strong>NULL Ordering:</strong> YDB/YQL follows standard SQL NULL ordering behavior:
```

Comment:
Suggestion: Let's also add docs on `null` ordering to `OrderExpression.SortOrder`. The in-memory implementation also orders NULLS FIRST, so the doc can mostly be copied straight away.